src/multi/bmp180.cpp

```diff
@@ -31,84 +35,76 @@ auto uncompensated_output_data_register()
   return std::to_array<hal::byte>({ 0xF6 });
 }
 
-auto soft_reset_register()
+// Helper function that creates an array containing the address and message for
+// sensor reset.
+auto soft_reset_config()
 {
-  return std::to_array<hal::byte>({ 0xE0 });
+  hal::byte reset_address = 0xE0;
+  hal::byte reset_setting = 0xB6;
```

Comment:
```suggestion
  constexpr hal::byte reset_address = 0xE0;
  constexpr hal::byte reset_setting = 0xB6;
```
Make all constants constexpr when you can.